test/transaction.ts

```diff
@@ -1022,7 +1039,7 @@ describe('Transaction', () => {
       it('should accept gaxOptions', done => {
         const gaxOptions = {};
         transaction.request = config => {
-          assert.strictEqual(config.gaxOpts, gaxOptions);
+          assert.deepStrictEqual(config.gaxOpts, gaxOptions);
```

Comment:
```suggestion
          assert.strictEqual(config.gaxOpts, gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `gaxOptions`.